multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/CheckServicesToDeleteStep.java

```diff
@@ -30,35 +26,19 @@
     private ApplicationConfiguration applicationConfiguration;
 
     @Override
-    protected Set<CloudServiceInstanceExtended> getExistingServicesToProcess(ProcessContext context) {
+    protected List<CloudServiceInstanceExtended> getExistingServiceInProgress(ProcessContext context) {
         List<String> servicesToDelete = context.getVariable(Variables.SERVICES_TO_DELETE);
         CloudControllerClient client = context.getControllerClient();
         int maxParallelThreads = getMaxParallelThreads(servicesToDelete);
-        return ForkJoinPoolUtil.execute(maxParallelThreads, () ->  getExistingServices(client, servicesToDelete));
+        return ForkJoinPoolUtil.execute(maxParallelThreads, () ->  getExistingServicesInProgressInternal(client, servicesToDelete));
```

Comment:
Either this name or `doGetExistingServicesInProgress`. Both are good but since the name is already long, why make it longer :D